en/docs/quick-starts/webapp-custom-sample.md

```diff
@@ -0,0 +1,49 @@
+#Enable Login for a Sample Custom Web Application
+This page guides you through integrating custom inbound authenticator with WSO2 Identity Server.
+
+----
+If you have your own application, click the button below.
+
+<a class="samplebtn_a" href="../../guides/login/integrate-custom-protocol" rel="nofollow noopener">I have my own application</a>
+
+----
+
+{!fragments/deploying-sample-apps.md!}
+
+----
+## Download the sample
+To be able to deploy a WSO2 Identity Server sample, you need to download it onto your machine first.
+
+Follow the instructions below to download the sample from GitHub. 
+Please refer ['Write a Custom Inbound Protocol'](insertlink) section for the implementation details.
+
+1. Navigate to [WSO2 Identity Server Samples](insertlink).
+
+2. [Download](inserlink) the TODO.war file and TODO-authenticator.jar from the latest release assets.
+
+---
+## Deploy the inbound authenticator
+
+ - Copy the TODO-authenticator.jar to dropins directory (i.e. /repository/components/dropins)
```

Comment:
to dropins directory --> to the `<IS_HOME>/repository/components/dropins` directory.